src/compiler/scala/tools/nsc/ast/TreeInfo.scala

```diff
@@ -97,4 +97,12 @@ abstract class TreeInfo extends scala.reflect.internal.TreeInfo {
     case DocDef(_, definition) => isPureDef(definition)
     case _ => super.isPureDef(tree)
   }
+
+  override def firstConstructor(stats: List[Tree]): Tree = {
+    val unwrapped = stats map {
+      case DocDef(_, defn) => defn
```

Comment:
good catch! It's probably rare, but from all the bugs I've seen so far, I wouldn't be surprised if there is a `/**comment about */ /** a comment*/` somewhere out there